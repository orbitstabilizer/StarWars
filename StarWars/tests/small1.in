3
Ryndellia REPUBLIC
Duro REPUBLIC
Mygeeto SEPARATISTS
10
Sith Dooku 200 50 125
Officer Grievous COMMAND 9
Jedi Windu 350 80 70
Officer Trent COMMAND 3
Officer Urhal GUNNERY 2
Officer Billaba PILOTING 1
Jedi Kenobi 100 100 100
Jedi Skywalker 10 120 20
Officer Ashoka PILOTING 3
Officer Rex COMMAND 3
3
SeparatistBattleship Malevolence 3 5
2 1 2
RepublicCruiser Halleck 2 1
4 3 4 5 6
RepublicCruiser Negotiator 3 1
4 7 8 9 10
15
50 2
50 2
50 4
50 4
51 1 Armament 10
51 2 Shield 20
41 10 3
40 10 2
20 2 3 4
10 1 2
11 3
20 1 1 5
20 3 1 1
11 1
30 3