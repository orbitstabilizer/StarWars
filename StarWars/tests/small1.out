Warship Negotiator in (Ryndellia, 5)
Kenobi 3282
Warship Malevolence is destroyed by Negotiator in (Ryndellia,5)
Warship Halleck is destroyed by Malevolence in (Mygeeto,4)
Sith Dooku is free
886
Jedi Windu is killed by Dooku
605
Jedi Kenobi is in Negotiator
500
Jedi Skywalker is in Negotiator
390
Officer Grievous is imprisoned
COMMAND 10
Officer Trent is killed by Dooku
COMMAND 5
Officer Ashoka is in Negotiator
PILOTING 3
Officer Rex is killed by Dooku
COMMAND 3
Officer Urhal is killed by Dooku
GUNNERY 2
Officer Billaba is killed by Dooku
PILOTING 1
